ffv1.md

```diff
@@ -1111,7 +1111,7 @@ If state_transition_delta is not present in the FFV1 bitstream, all Range coder
 | Other | reserved for future use         | reserved for future use         | reserved for future use         |
 
 Restrictions:  
-If `colorspace_type` is 1, `chroma_planes` MUST be 1, `h_chroma_subsample` MUST be 1, `v_chroma_subsample` MUST be 1.  
+If `colorspace_type` is 1, then `chroma_planes` MUST be 1, `h_chroma_subsample` MUST be 1, and `v_chroma_subsample` MUST be 1.  
```

Comment:
I missed this one in the patch about chroma_planes, but should actually be `log2_h_chroma_subsample` and `log2_v_chroma_subsample`

Reply:
not `log2(h_chroma_subsample)`?

Reply:
no, we changed it.
check `Parameters( )` pseudo-code `log2_h_chroma_subsample` definition part

Reply:
fixed in a second commit, thx for catching this.